Gruntfile.js

```diff
@@ -231,100 +273,170 @@ module.exports = function(grunt) {
+      },
+    },
+  })
 
   /* Tasks
   ================================================= */
 
-  grunt.registerTask('default', ['css', 'html']);
-
-  // Assets
-  grunt.registerTask('html', ['assemble', 'inline', 'premailer:txt', 'replace', 'prettify']);
-  grunt.registerTask('css', ['sass', 'autoprefixer']);
-  grunt.registerTask('images-ftp', ['ftp-deploy']);
-  grunt.registerTask('images-s3', ['s3']);
+  grunt.registerTask('default', [
+    'clean:dist',
+    'css',
+    'postmark-templates',
+    'generic-templates',
+  ])
+
+  /**
+   * Compiles Postmark compatible layouts and templates into separate files.
+   * Ideal if Postmark is your email service provider.
+   * CSS is not inlined since Postmark handles this for you.
+   */
+  grunt.registerTask('postmark-templates', [
+    'copy:postmarkTemplates', // Copy Handlebars templates to dist folder
+    'copy:postmarkLayouts', // Copy Handlebars layouts to dist folder
+    'replace:postmarkPlaceholder', // Replace handlebars {{body}} with Postmark content placeholder
+    'replace:postmarkConfig', // Remove handlebars config from layouts
+    'assemble:postmark', // Compile handlebars templates and layouts. Mainly used for loading partials into the templates.
+    'inline:postmark', // Load external CSS into layouts
+    'replace:styleBlock', // Add properties to style block
+    'prettify:postmark', // Format layout HTML
+    'premailer:txt', // Generate text versions
+    'replace:variableSyntax', // Clean up variable syntax in text versions
+    'clean:hbs', // Remove handlebars files
+  ])
+
+  /**
+   * Compiles templates and layouts to dist/compiled.
+   * Each template file includes the combined layout with inlined CSS.
+   * Ideal if you don't use Postmark as an email service provider.
+   */
+  grunt.registerTask('generic-templates', [
+    'assemble:generic', // Compile handlebars templates to HTML
+    'inline:generic', // Load external CSS into templates
+    'premailer', // Inline CSS and generate text version
+    'replace:variableSyntax', // Clean up variable syntax in text versions
+    'replace:styleBlock', // Add properties to style block
+    'prettify:generic', // Format HTML
+  ])
+
+  grunt.registerTask('css', ['sass', 'autoprefixer'])
+  grunt.registerTask('images-ftp', ['ftp-deploy'])
+  grunt.registerTask('images-s3', ['s3'])
+  grunt.registerTask('deploy', ['default', 'shell:postmarkPush'])
 
   // Testing
-  grunt.registerTask('spam', ['spamcheck']);
-  grunt.registerTask('litmus', ['testBuild', 'postmark:litmus']);
-  grunt.registerTask('flood', ['testBuild', 'postmark:flood']);
-
-  // Before sending tests via Postmark, ensure that test builds with inlined CSS are generated
-  grunt.registerTask('testBuild', ['default', 'copy:testTemplates', 'premailer:html']);
-};
+  grunt.registerTask('spam', ['spamcheck'])
+  grunt.registerTask('litmus', ['default', 'postmark:litmus'])
+  grunt.registerTask('flood', ['default', 'postmark:flood'])
```

Comment:
When I run this I receive the compiled emails but also get a bunch of emails with just the body content, but no layout. I think it’s just sending all of the html files in the list folder, but should we only be sending the compiled templates?